jenkins/bootstrap.py

```diff
@@ -818,10 +872,16 @@ def parse_args(arguments=None):
     parser.add_argument('--root', default='.', help='Root dir to work with')
     parser.add_argument(
         '--timeout', type=float, default=0, help='Timeout in minutes if set')
-    parser.add_argument('--pull',
-        help='PR, or list of PR:sha pairs like master:abcd,12:ef12,45:ff65')
-    parser.add_argument('--branch', help='Checkout the following branch')
-    parser.add_argument('--repo', help='The repository to fetch from')
+    parser.add_argument(
+        '--pull',
+        help='Deprecated, use --repo=k8s.io/foo=master:abcd,12:ef12,45:ff65')
+    parser.add_argument(
+        '--branch',
+        help='Deprecated, use --repo=k8s.io/foo=master')
+    parser.add_argument(
+        '--repo',
+        action='append',
+        help='Fetch the specified repositories, with the first one considered primary')
```

Comment:
where is the first repo made primary? it seems like parse_repos doesn't do any sort of ordering priority